plugins/rich-editor/src/scripts/editor/richEditorFormClasses.ts

```diff
@@ -23,7 +23,7 @@ import { richEditorVariables } from "@rich-editor/editor/richEditorVariables";
 
 export const richEditorFormClasses = useThemeCache((legacyMode: boolean = false) => {
     const globalVars = globalVariables();
-    const headerVars = titleBarVariables();
+    const titleBarVars = titleBarVariables();
```

Comment:
I'm pretty sure this file is just deleted at this point. It's been moved into KB as `editorFormStyles`, but probably got caught up in those merge conflicts a couple weeks ago. Could we just delete this file?